collectors/python.d.plugin/dns_query_time/dns_query_time.chart.py

```diff
@@ -89,13 +84,15 @@ def dns_req(ns, t, q):
         request = dns.message.make_query(domain, dns.rdatatype.A)
 
         try:
-            dns_start = time()
-            dns.query.udp(request, ns, timeout=t)
-            dns_end = time()
-            query_time = round((dns_end - dns_start) * 1000)
-            q.put({'_'.join(['ns', ns.replace('.', '_')]): query_time})
+            resp = dns.query.udp(request, ns, timeout=t)
+            if (resp.rcode() == dns.rcode.NOERROR and resp.answer):
+                query_time = resp.time * 1000
```

Comment:
please change to `100000`

resp.time in seconds, we convert it to ms 

* 1000

since we add `100` as divisor in chart, we need to multiply by 1000 * 100 (100000)

you will see x100 numbers in debug, but they will be ok on the dashboard